### ISSUE TITLE
Crouton Fedora support

### ISSUE THREAD

**Issue by nmilosev** [divx118/crouton-packages#38]
Hey @divx118 this isn't really an issue, just wanted to thank you for packaging these packages, and to let you know that they are working perfectly on my fork of Crouton for Fedora. I've managed to install VirtualBox 5.0 and even Docker (with docker-machine) fairly easily. 

I wrote a few blog posts about how it can be done, which could be interesting for other people trying to do the same thing:

https://nmilosev.svbtle.com/virtualbox-and-crouton-fedora
https://nmilosev.svbtle.com/docker-on-crouton-fedora-24

Sadly, I too ran into issues with bridged adapters rebooting my Chromebook and host-only adapters not working. Still, NAT with port forwarding works like a charm. :)

Thank you again for your work.

Nemanja

**Comment by divx118** [comment]
@nmilosev Yes, known issue with the network adapter setting causing a reboot. If I have some time I will look into it, however I am afraid it will need our own build kernel to solve it. Which would be better then to use legacy boot for example from a SD card or USB stick.

BTW thanks for the blog posts and continuing the work on fedora. Sadly as you may have noticed it will be a real pain to get it pulled into the crouton branch without making too many changes to the current crouton code. At least that is one of the reasons I didn't continue the work on it.  

for reference https://github.com/divx118/crouton-packages/issues/37

**Comment by nmilosev** [comment]
Yes, Fedora is gonna be hard to merge fully. It's fine as it is for now though, I think. :)

Thanks again for your work, it was really helpful.

Cheers!
